packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx

```diff
@@ -73,17 +78,19 @@ function MessageRoomDetailsDialog({
     const org_id = organization.user.org_id;
 
     const fetchData = async () => {
-      const res = await fetch(
-        `https://chat.zuri.chat/api/v1/org/${org_id}/rooms/${room_id}`
-      );
-
-      if (!res.ok) {
-        throw Error("Could not fetch data");
-      }
-
-      const data = await res.json();
-      setRoomData(data);
-      setDescription(data.data.description);
+      await axios
+        .get(`https://chat.zuri.chat/api/v1/org/${org_id}/rooms/${room_id}`)
+        .then(res => {
+          setRoomData(res.data);
+          setDescription(res.data.data.description);
+        })
+        .catch(e => {
+          setErrorModal("Could not fetch description");
+
+          setTimeout(() => {
+            setErrorModal("");
+          }, [4000]);
+        });
```

Comment:
clear your timeout.

Reply:
Is isFetched not going to clean it up? 👀